Minio/Minio.csproj

```diff
@@ -2,7 +2,7 @@
 	<PropertyGroup>
 		<AssemblyName>Minio</AssemblyName>
 		<RootNamespace>Minio</RootNamespace>
-		<TargetFrameworks>net6.0;net7.0;netstandard2.0</TargetFrameworks>
+		<TargetFrameworks>net6.0;net7.0;net8.0;netstandard2.0</TargetFrameworks>
```

Comment:
You added `net8.0` here instead of replacing `net7.0` like you've done it above in `Minio.Functional.Tests/Minio.Functional.Tests.csproj` file.
We have to be consistent all over this PR.
So, my vote is to remove `net7.0` from all `TargetFrameworks` definitions.

Reply:
I agree. Removed net7 support.